server/index.js

```diff
@@ -15,6 +15,7 @@ app.listen(port, () => {
 const allowedOrigins = [
   'http://localhost:3000',
   'http://localhost:5500',
+  'https://fec-overview.s3-us-west-2.amazonaws.com'
```

Comment:
It doesn't appear other people have access to your S3, I think you have to give us individual permissions with our emails.